CHANGELOG.md

```diff
@@ -25,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `fantasy` font renamed to `fantasy1`
 - `upside_down` font renamed to `upside_down1`
 - `sep` parameter added to `text2art`, `tprint`, `tsave` and `set_default` functions
+- Default line separator changed from `\r\n` to `\n`
```

Comment:
Isn't better if we add this line into `[Unreleased]` section?

Reply:
> Isn't better if we add this line into `[Unreleased]` section?

It's something related to previous version